mod-codex-ekb/src/test/resources/domain/codexekb/basic-junit.feature

```diff
@@ -3,17 +3,20 @@ Feature: mod-codex-ekb integration tests
   Background:
     * url baseUrl
     * table modules
-      | name              |
-      | 'mod-login'       |
-      | 'mod-permissions' |
-      | 'mod-codex-ekb'   |
+      | name                |
+      | 'mod-login'         |
+      | 'mod-permissions'   |
+      | 'mod-codex-ekb'     |
+      | 'mod-codex-mux'     |
```

Comment:
Why this module is needed there?

Reply:
mod-codex-ekb depends on this modules.
This has been agreed with Oleksandr Dekin

Reply:
It's strange because in the module descriptor there is no dependency for the mux module
![image](https://user-images.githubusercontent.com/5904625/128873730-33e51b34-16cc-4737-8545-c22a33056042.png)